.prettierrc

```diff
@@ -1,12 +1,12 @@
-
 {
   "arrowParens": "avoid",
   "bracketSpacing": true,
-  "endOfLine":"auto",
+  "endOfLine": "auto",
   "printWidth": 120,
-  "singleQuote": false,
+  "singleQuote": true,
```

Comment:
This can not work.

This is prettier for solidity files only and solidity does not support single quote strings